src/redux/books/books.js

```diff
@@ -0,0 +1,39 @@
+// Actions
+const ADD = 'bookstore-1/books/ADD';
+const UPDATE = 'bookstore-1/books/UPDATE';
+const DELETE = 'bookstore-1/books/DELETE';
+const LOAD = 'bookstore-1/books/LOAD';
+
+// Reducer
+export const reducer = (state = [], action = {}) => {
+  switch (action.type) {
+    case ADD: {
+      return state;
+    }
+
+    case UPDATE: {
+      return state;
+    }
+
+    case DELETE: {
+      return state;
+    }
```

Comment:
- [ ] in case of action type `delete` the function should use ES6 `filter()` method to create a new array, which will not contain the book you want to remove from the store. Kindly update the returned value 👍